src/test/SubscriptionEmailTemplate.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
```

Comment:
this is already imported in the `setup.ts` file

Reply:
fixed